source/optional_features/use_your_own_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: Use your own error pages 
+weight: 135
+---
+
+# Use your own payment failure pages
+
+You can [contact us](/support_contact_and_more_information/#contact-us) to set
+this feature up for your service. 
+
+Terminal, unsuccessful payment states are:
+
+* payment declined
+* payment cancelled
+* payment expired
+* error (eg technical error)
+
+If you use this feature, when a payment reaches one of these states, GOV.UK
+Pay will immediately redirect the user to the return URL (`return_url`). You
+can set the `return_url` manually for your service, in the response to the
+`POST /v1/payments` API call. 
+
```

Comment:
oh, i thought it was set in the request rather than response? @alexbishop1 can confirm

Reply:
Yeah it will be, this is just a mistake. I'll wait for Alex' reply in any case.